contributor_path.rst

```diff
@@ -0,0 +1,212 @@
+****************
+Contributor path
+****************
+
+..
+
+  "There is no favorable wind for the sailor who doesn’t know where to go."
+
+  -- Seneca
+
+This guide describes a contributor path in scope of the Ansible project from the early beginning, through all the stages, to becoming a leader shaping the project's future and the future of automation in the IT world in general.
+
+This path is not a policy but rather a suite of recommendations based on experience of successful contributors.
+
+The goal of the path, first of all, is to make your activity more determined and efficient through understanding of what you can achieve and how. You can use this path as a roadmap for your long-term participation.
+
+Any contribution to the project, even a small one, is very welcome and valuable. When you contribute regularly, your proficiency and judgement in the related area are increasing fast and, along with this, the importance of your presence in the project.
+
+Regular contribution makes you a member of the community and leads to respect of other community members. Do you respect people who give good to others? Do you respect people who help you as a user whose feature request was implemented or who help you as a programmer to finish your pull request?
```

Comment:
"give good" doesn't work here. What's the message you want to get across? I could see:

- people who help others
- people who give their time to others
- people who respect others

There's probably more, but I'm guessing at what you're going for :)

Reply:
@GregSutcliffe I tried to use something short and meaningful but failed - this is really very broad, do you have any ideas?

Reply:
I changed it to "any improvements", is it better?

Reply:
changed to "any addition or positive change" almost like @acozine suggested in another PR. How does it feel now?

Reply:
@Andersson007 we've misunderstood each other - I'me referring to the sentence "Do you respect people who give good to others?" on line 19, your last commits seem to change line 34?

Reply:
@GregSutcliffe ah.. hehe:) Ok, I'll change it. Anyway which one sounds better `any good` or `any addition or positive change` ?

Reply:
For line 34? I think `positive change` is better, leave it as it is now. I think you've been a bit harsh on line 19 though, most of it is gone now - I just wanted to fix "give good", the overall message was fine :)

Reply:
Yeah, for line 34. Help of native speaker in such things is much appreciated, thanks!
I decided to get rid of those sentences on line 19 because the idea is obvious without them i think.